File: java/programs/threedigit.java
import java.util.Scanner;
public class threedigit {
    public static void main(String[] args){
        Scanner sc =new Scanner( System.in);
        System.out.println("enter a number:");
        int n =sc.nextInt();
        if(n%3==0 ||n%5==0){
          if(n%15!=0){
            System.out.println(" this is divisible by 3 or 5 but not 15");
          }
          else{
            System.out.println("not matching required condition ");
          }
        }      


    }
}

File: c language/control structure/threedigit.c
#include <stdio.h>
int main(){
    int n;
    printf("enter anumber:");
    scanf("%d",&n);
    if(n%5==0|| n%3==0){
        printf("the no is divisible by 3or 5");

    }
    else{
        printf("not divisible");
    }
    return 0;    
}